common/src/main/scala/explore/common/SSOClient.scala

```diff
@@ -162,9 +134,7 @@ object SSOClient {
       )
       Timer[F].sleep(sleepTime / 10)
```

Comment:
This is good for testing, eventually we want to remove the `/10`

Reply:
I'll try to see if it's easy to read it from the environment configuration.